src/lib/plugin_apis/lvm.api

```diff
@@ -1055,6 +1055,22 @@ gboolean bd_lvm_lvresize (const gchar *vg_name, const gchar *lv_name, guint64 si
  */
 gboolean bd_lvm_lvactivate (const gchar *vg_name, const gchar *lv_name, gboolean ignore_skip, const BDExtraArg **extra, GError **error);
 
+/**
+ * bd_lvm_lvactivate_shared:
+ * @vg_name: name of the VG containing the to-be-activated LV
+ * @lv_name: name of the to-be-activated LV
+ * @ignore_skip: whether to ignore the skip flag or not
+ * @shared: whether to activate the LV in shared mode
```

Comment:
Isn't this argument somewhat misleading since you have `_shared` in the function name already?

Reply:
I am open to suggestions for a better name. To me this means "new version of the function with `shared` agument".

Reply:
I understand, but with the new function having the `shared` argument I can just call it and pass a "shared" variable to it instead of using a condition.